src/setup-utils/SwiftDocCRenderi18n.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2023 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import VueI18n from 'vue-i18n';
```

Comment:
Maybe move this to the `/plugins` dir?

Reply:
Well, I implemented it in the same way as `SwiftDocCRenderRouter` is. What part of the code would you move to the plugins folder?